### PR TITLE
Allow control of verbosity for CLI and set default level to warning.

### PR DESCRIPTION
**Pull request by csadorf**

### CODE REVIEW

File: aiidalab/__main__.py [hunk 1]
```diff
@@ -72,8 +69,9 @@ def _list_apps(apps_path):
 
 @click.group()
 @click.version_option(version=__version__, prog_name="AiiDAlab")
-def cli():
-    pass
+@click.option("-v", "--verbose", count=True)
+def cli(verbose):
+    logging.basicConfig(level=logging.WARNING - 10 * verbose)
```
Comment (yakutovicha) [inline aiidalab/__main__.py:74]: thanks, @csadorf. Just one question: what happens if the level value is negative?

**Review by yakutovicha — APPROVED** [review]
One question, otherwise good to go.

File: aiidalab/__main__.py (hunk 1 above)
Comment (csadorf) [reply aiidalab/__main__.py:74]: Just ran the following minimal demo:
```python
import logging

logging.basicConfig(level=-1)
logger = logging.getLogger('test')

logger.info("hello")
```
and it's fine, however, I suggest that we simply safeguard against potential undefined behavior with:
```suggestion
    logging.basicConfig(level=max(0, logging.WARNING - 10 * verbose))
```

**Comment by yakutovicha** [comment]
@csadorf do you want to merge this?

**Pull request merged**